### PR TITLE
remove bugs and warnings, search, helpful, and some buttons are working

### PR DESCRIPTION
**Pull request by lucky89nana**
@arangotang @r-snow @kennytran95 
Can you review the code when you have time? 
Thanks!

### CODE REVIEW

File: src/components/questions_answers/QnA_Components/AddAnswer.jsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  //     e.stopPropagation();
+  //     return;
+  //   }
+  //   setShowModal(false);
+  // }
+
+  return (
+    <div>
+      <div>
+        <div>
```
Comment (arangotang) [inline src/components/questions_answers/QnA_Components/AddAnswer.jsx]: Screen Shot 2022-05-22 at 11 29 04 AM

Seeing we have two extra divs surrounding the content of this component. Would it be ok to remove them?

File: src/components/questions_answers/QnA_Components/AddAnswer.jsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+          <div>
+            <h5>Submit Your Answer: {question.question_body}</h5>
+            <button
+              type="button"
+              onClick={() => setShowModal(false)}
+              data-dismiss="modal"
+              aria-label="Close"
+            />
```
Comment (arangotang) [inline src/components/questions_answers/QnA_Components/AddAnswer.jsx]: This button seems to be doing the same thing as the button on lines 100-106. Can this be removed?

File: src/components/questions_answers/QnA_Components/MoreAnswers.jsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+          >
+            LOAD MORE ANSWERS
+          </button>
+        </div>
+      ) : (
+        <div>
+          {answerArr.slice(2).map((answer) => (
+            <AnswerList answer={answer} key={answer.id} />
+          ))}
+          <div>
```
Comment (arangotang) [inline src/components/questions_answers/QnA_Components/MoreAnswers.jsx]: The only child for this div is the button, are we ok to remove this?

File: src/components/questions_answers/QnA_Components/MoreQuestions.jsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+          >
+            More Answered Questions
+          </button>
+        </div>
+      ) : (
+        <div>
+          {questions.slice(4).map((question) => (
+            <Question question={question} key={question.question_id} />
+          ))}
+          <div>
```
Comment (arangotang) [inline src/components/questions_answers/QnA_Components/MoreQuestions.jsx]: Same question as above on MoreAnswers

**Review by arangotang — CHANGES_REQUESTED** [review]
Hey @lucky89nana , looking really good -- nice to see this getting built out.

I left a few questions in the review; can you take a look and advise on them? Thanks

File: src/components/questions_answers/QnA_Components/MoreQuestions.jsx (hunk 4 above)
Comment (lucky89nana) [reply src/components/questions_answers/QnA_Components/MoreQuestions.jsx]: hi @arangotang , thank you for reviewing, I have made some changes and added some basic css, hope it look better now. Thanks!

**Comment by lucky89nana** [comment]
hi @arangotang , thank you for reviewing, I have made some changes and added some basic css, hope it look better now. Thanks!

File: src/components/questions_answers/QnA_Components/AnswerPhotos.jsx [hunk 5]
```diff
@@ -1,3 +1,45 @@
...
+        role="presentation"
+        onClick={handlePhotoClick}
+        onKeyPress={handlePhotoClick}
+      />
+      {openModal ? (
+        <div className="answer-photo-modal ">
+          <button
+            className="enlarged-answer-photo-close"
+            type="button"
+            onClick={handlePhotoClick}
```
Comment (arangotang) [inline src/components/questions_answers/QnA_Components/AnswerPhotos.jsx:27]: To make the 'x' button more visible here, would recommend adding the below styling:
`style={{
              position: 'absolute',
              top: '1em',
              right: '1em',
            }}
`

**Review by arangotang — CHANGES_REQUESTED** [review]
Everything looks really good; love the styling and modals.

I just made one quick comment on AnswerPhotos re: the modal 'x' button, lmk what you think!

**Review by arangotang — APPROVED** [review]
:thumbsup:

**Pull request merged**